packages/RichTextEditing-Core.package/RichTextDocument.class/instance/sortRunsByPriority.st

```diff
@@ -5,6 +5,6 @@ sortRunsByPriority
 	self text runs: (self textRuns copy collect: [:attributes | 
 		structureCopy := attributes copy select: [:attribute | attribute isTextStructure].
 		noStructureCopy :=  attributes copy reject: [:attribute | attribute isTextStructure].
-		structureCopy := structureCopy copy sortBy: [:a :b | 
+		structureCopy := structureCopy sorted: [:a :b | 
```

Comment:
Why?

Reply:
Because sortBy: is deprecated and the message is therefore not understood in the CI images any more.